Havit.Blazor.Documentation/Havit.Blazor.Documentation.csproj

```diff
@@ -16,6 +16,7 @@
 	<ItemGroup>
 		<PackageReference Include="LoxSmoke.DocXml" />
 		<PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" />
+		<PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" />
```

Comment:
> This package has been deprecated as it is legacy and is no longer maintained.

Reply:
@Harvey1214 See the commit added. The solution is a little tricky as we don't want to add any extra dependency to WASM (downloaded to browser).